### PR TITLE
Relax UI chart limits

### PR DESCRIPTION
**Pull request by scavnickyj**
---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Comment by scavnickyj** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2625/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18780/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2135/ : UNSTABLE in 24m 51s

**Comment by scavnickyj** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18781/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20481/ : SUCCESS in 9m 08s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2626/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18782/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2136/ : SUCCESS in 24m 32s

**Comment by scavnickyj** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18783/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20483/ : SUCCESS in 9m 01s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2627/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18784/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2137/ : SUCCESS in 27m 03s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18785/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20484/ : SUCCESS in 9m 11s

File: libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartLimits.ts [hunk 1]
```diff
@@ -80,6 +85,55 @@ function getChartLimits(type: string): IChartLimits {
     }
 }
 
+function getSoftChartLimits(type: string): IChartLimits {
```
Comment (kandl) [inline libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartLimits.ts:88]: Should be `type: VisualizationTypes` or similar for stronger typing?

File: libs/sdk-ui-kit/src/Icon/icons/LegendMenu.tsx [hunk 2]
```diff
@@ -16,7 +16,11 @@ export const LegendMenu: React.FC<IIconProps> = ({ color, className, width, heig
             xmlns="http://www.w3.org/2000/svg"
         >
             <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
-                <g transform="translate(-5.500000, -6.000000)" fill={color ?? "#6D7680"}>
+                <g
+                    id="LegendIcoContent"
```
Comment (kandl) [inline libs/sdk-ui-kit/src/Icon/icons/LegendMenu.tsx:20]: Should it be `id`? If I have multiple charts, id wont be unique?

Comment (scavnickyj) [reply libs/sdk-ui-kit/src/Icon/icons/LegendMenu.tsx:20]: This was there even before but it was deleted for some reason. It was used for properly selecting the inner part and changing its color by css. Was not working without it. Proper css specificity should make it safe.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18786/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20485/ : SUCCESS in 9m 08s

**Comment by scavnickyj** [comment]
extended test - backstop

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2628/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/2099/ : SUCCESS in 1m 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20487/ : SUCCESS in 9m 03s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1899/ : UNSTABLE in 1m 43s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1881/ : SUCCESS in 6m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5396/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/2100/ : FAILURE in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20488/ : SUCCESS in 9m 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1900/ : SUCCESS in 9m 38s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1882/ : UNSTABLE in 8m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5397/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/2101/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20489/ : SUCCESS in 9m 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1901/ : SUCCESS in 12m 02s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1883/ : SUCCESS in 8m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5398/ : SUCCESS in 7s

**Pull request merged**